src/transformers/image_utils.py

```
@@ -239,6 +240,10 @@ def infer_channel_dimension_format(
         first_dim, last_dim = 0, 2
     elif image.ndim == 4:
         first_dim, last_dim = 1, 3
+    elif image.ndim == 2:
+        raise ValueError(
+            "must specify input_data_format = ChannelDimension.NONE and use supported image processor to use (height, width) image"
+        )
```

Why should we raise here instead of returning `ChannelDimension.NONE`?